src/views/routes/users.js

```diff
@@ -61,16 +66,66 @@ router.post('/login', async (req, res) => {
   }
 });
 
+router.get('/:username', async (req, res) => {
+  try {
+    const { username } = req.params;
+    await getUser({ username });
+
+    responseHandler.handleSuccess(res, success.CODE);
```

Comment:
Why does `GET /users/:username` simply return 200? I would expect that some data is returned to the client.

Reply:
At first, I was returning the username/email but I decided to not return any data to the client as it's not "needed" by my front-end component. Currently, the client side of my application does not use the data that we are getting back fro this end-point. However, I will include it for future use. But as I said, the data is not being used or expected by my front-end application

Reply:
Updated!